recipe/meta.yaml

```
@@ -13,29 +13,47 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }}  -m pip install . --no-deps --ignore-installed -vv
+  script: |
+# Replace deprecated SafeConfigParser with ConfigParser
+# Use perl for cross-platform compatibility
+    perl -pi -e 's/configparser\.SafeConfigParser/configparser.ConfigParser/g' versioneer.py
+    perl -pi -e 's/parser\.readfp/parser.read_file/g' versioneer.py
+    {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
+  build:
+    - perl  # [not win]
   host:
-    - python >=3.5
+    - python
     - pip
     - jinja2
+    - wheel
   run:
-    - python >=3.5
+    - python
     - intake >=0.3
     - pandas
     - sqlalchemy
```

```suggestion
    - sqlalchemy >2
```
From https://github.com/intake/intake-sql/blob/052cde14156fa2077655d0fc6115e11c2e9f4174/requirements.txt#L3